python/lsst/analysis/tools/atools/photometry.py

```diff
@@ -89,6 +89,57 @@ def setDefaults(self):
         }
 
 
+class PsfCModelSkyPlot(AnalysisTool):
```

Comment:
Would be nice to add a one-line docstring; one can pretty much figure it out from the class name but it's nice to have it in text.